### ISSUE TITLE
Indentation when using comments in nested Sass maps

### ISSUE THREAD

**Issue by backflip** [stylelint/stylelint#3630]
> Clearly describe the bug

When using a comment within a nested Sass map, in incorrect indentation is expected.

> Which rule, if any, is the bug related to?

`indentation`

> What CSS is needed to reproduce the bug?

Breaking:
```scss
$size: (
  small: (
    width: 200 // hello
  ),
  medium: (
    width: 400
  )
);
```

Working fine (further indenting second block):
```scss
$size: (
  small: (
    width: 200 // hello
  ),
    medium: (
      width: 400
    )
  );
```

Working fine, too (comment removed):
```scss
$size: (
  small: (
    width: 200
  ),
  medium: (
    width: 400
  )
);
```

> What stylelint configuration is needed to reproduce the bug?

Tested with default on https://stylelint.io/demo/

> Which version of stylelint are you using?

Whatever https://stylelint.io/demo/ is using

> How are you running stylelint: CLI, PostCSS plugin, Node API?

Demo website

> Does the bug relate to non-standard syntax (e.g. SCSS, Less etc.)?

Yes, it's related to nested Sass maps.

> What did you expect to happen?

No errors to be flagged.

> What actually happened (e.g. what warnings or errors did you get)?

The following errors were flagged:

```
5:3 error Expected indentation of 4 spaces (indentation)
6:5 error Expected indentation of 6 spaces (indentation)
7:3 error Expected indentation of 4 spaces (indentation)
8:1 error Expected indentation of 2 spaces (indentation)
```

**Comment by backflip** [comment]
Sorry, found a duplicate in the meantime: https://github.com/stylelint/stylelint/issues/2946
